Client/Frontend/Strings.swift

```diff
@@ -3821,15 +3821,15 @@ extension String {
             tableName: "Shopping",
             value: "How we determine review quality",
             comment: "Title of the 'How we determine review quality' card displayed in the shopping review quality bottom sheet.")
-        public static let ReviewQualityCardExpandedAccessibilityLabel = MZLocalizedString(
+        public static let ReviewQualityCardExpandAccessibilityLabel = MZLocalizedString(
             key: "Shopping.ReviewQualityCard.Expanded.AccessibilityLabel.v120",
```

Comment:
The keys for the expand/collapse a11y labels are not matching its content anymore.

Reply:
"Shopping.ReviewQualityCard.Expand.AccessibilityLabel.v120" would be ok?

Reply:
That works.